File: 2042-maximum-product-difference-between-two-pairs/maximum-product-difference-between-two-pairs.py
class Solution:
    def maxProductDifference(self, nums: List[int]) -> int:
        n = len(nums)
        nums = sorted(nums)

        a = nums[0]
        b = nums[1]
        c = nums[n - 1]
        d = nums[n - 2]

        ans = ((c * d) - (a * b))

        return ans

File: 2042-maximum-product-difference-between-two-pairs/maximum-product-difference-between-two-pairs.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxProductDifference = function(nums)
{
    nums.sort(function(a, b){return a - b})
    let n = nums.length

    let a = nums[0]
    let b = nums[1]
    let c = nums[n - 2]
    let d = nums[n - 1]

    let ans = (c * d) - (a * b)

    return ans
};